18 Normalizr/src/schemas/studentSchema.ts

```diff
@@ -0,0 +1,7 @@
+import { schema } from 'normalizr';
+import { countrySchema } from './countrySchema';
+
+export const studentSchema = new schema.Entity('students', {
+  country: countrySchema
```

Comment:
This is not true. Current API returns countryId, not a nested object. What I have understood so far is that this line would be helpful to denormalize country information from students, but this is not going to happen. If student gets a country object just for the UI, it doesn't apply here, and probably we would like to stop adding that country to the student object, instead you can look up the country from the Id when needed. Thumbs down to normalizr here, to extract nested object, I see value on it to convert countries array to a dictionary.